analytic_engine/src/sst/parquet/hybrid.rs

```diff
@@ -87,23 +87,24 @@ impl ArrayHandle {
 #[derive(Debug)]
 struct TsidBatch {
     non_collapsible_col_values: Vec<String>,
-    collapsible_col_arrays: Vec<ArrayHandle>,
+    // record_batch_idx -> ArrayHandle
+    // Store collapsible data in multi record batch.
+    collapsible_col_arrays: BTreeMap<usize, Vec<ArrayHandle>>,
```

Comment:
This structure may not very match in this situation. I original thought of this data structure:
```
HashMap<ArrayRef, Vec<SliceArg>>
```
In this way, ArrayRef is only used once for one record batch, however in current design, ArrayRef will be cloned multiple times. Take an example:


<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-right" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-right">tsid</th>
<th scope="col" class="org-left">value</th>
</tr>
</thead>

<tbody>
<tr>
<td class="org-right">100</td>
<td class="org-left">xx</td>
</tr>


<tr>
<td class="org-right">100</td>
<td class="org-left">xx</td>
</tr>


<tr>
<td class="org-right">200</td>
<td class="org-left">xx</td>
</tr>


<tr>
<td class="org-right">100</td>
<td class="org-left">xx</td>
</tr>


<tr>
<td class="org-right">100</td>
<td class="org-left">xx</td>
</tr>
</tbody>
</table>

In record batch above, TsidBatch for 100 will contains two ArrayRef.
```
1 -> vec![(ArrayRef, (0,1)), (ArrayRef, (3,1))]
```

In my design, it will be 
```
ArrayRef -> vec![(0, 1), (3,1 )]
```